packages/api/src/controllers/playback.ts

```diff
@@ -19,33 +22,56 @@ type PlaybackInfo = {
     playbackPolicy?: Asset["playbackPolicy"] | Stream["playbackPolicy"];
     source: {
       // the only supported format is HLS for now
-      hrn: "HLS (TS)";
-      type: "html5/application/vnd.apple.mpegurl";
+      hrn: "HLS (TS)" | "MP4";
+      type: "html5/application/vnd.apple.mpegurl" | "html5/video/mp4";
```

Comment:
Can you check with Jaron if these would be the values used by mist as well?

Reply:
Confirmed as `"html5/video/mp4"` by Jaron! On `ws` instead would be `ws/video/mp4`